core/src/main/java/io/github/tcdl/api/Requester.java

```diff
@@ -23,7 +24,7 @@
  * response mechanism in case we create Requester with {@literal RequestOptions.waitForResponses => 0} and received Acknowledgement response
  * before RequestOptions.ackTimeout or RequestOptions.responseTimeout (takes max of two).
  *
- * Please note: RequestOptions.waitForResponses represent number of response messages  with {@link io.github.tcdl.messages.payload.Payload} set and in case we received
+ * Please note: RequestOptions.waitForResponses represent number of response messages  with {@link io.github.tcdl.api.message.payload.Payload} set and in case we received
```

Comment:
Maybe {@link Payload} will be better than {@link io.github.tcdl.api.message.payload.Payload}?